.github/workflows/Android-CI-Espresso.yml

```diff
@@ -0,0 +1,56 @@
+name: PullRequest
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  buildTest:
+    name: Build & Test JDK ${{ matrix.java_version }}
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        java_version: [1.8]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
+        with:
+          fetch-depth: 0
+      - name: Install JDK ${{ matrix.java_version }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java_version }}
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.0.7
+      - name: Build sdk
+        run: ./gradlew :sdk:assemble
+      - name: Build project
+        run: ./gradlew :hellocardboard-android:assembleDebug
+      - name: Run tests
+        run: ./gradlew test
+  Check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
+        with:
+          fetch-depth: 0
+      - name: Install JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.0.7
```

Comment:
I see that this configuration is taking a dependency on malinskiy/action-android/install-sdk@release/0.0.7, which is authored by another person. Is there a way to avoid doing this?

Reply:
Should I fork it and use my fork ?

Reply:
Based on what I could see [here](https://proandroiddev.com/how-to-github-actions-building-your-android-app-773779bcacab) and [here](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md), Java and Android are already installed in the runners offered by GitHub actions, so I'd say that we could ditch that dependency. I also think that given the fact that the SDK currently doesn't have any test, only adding the build test would suffice. Could you please perform this changes and check that they work properly? Thanks.

Reply:
I did it

Reply:
@jballoffet Hmm...I think this is counter productive as it allows for future errors that are completely unrelated to the code changed in the commit being tested by the CI. If the build depends on Java 1.8 and someone tries to build the same code a few years down the road and it breaks because the new runner defaults to Java 1.9, then this is non-deterministic behavior, which I thought you were not a fan of.

Reply:
Sorry for the long delay here. I think just using the pre installed Java and Android SDK version would suffice for now. Would you be so kind as to remove the ‘Install Android SDK step’ and check the CI works? Thanks.

Reply:
😄 You wait that long, till CI is broken.
I want to avoid PR bigger PR, but now I've to update build tools and Gradle as well to make it work https://github.com/hannesa2/cardboard/pull/4

Additionally I will not test a remove of install of SDK because it's waste of time.
To simplify I recommend `on: [pull_request]` but first I want to see this PR merged

If you want to test what's eliminate-able, just do it ! But I'm done here now